x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/index.ts

```diff
@@ -671,6 +699,24 @@ export const timelineWithReduxProperties = (
   pinnedEventsSaveObject: pinnedEvents,
 });
 
+export const resolvedTimelineWithReduxProperties = (
```

Comment:
I think we can remove this since we're using `timelineWithReduxProperties` directly here: https://github.com/elastic/kibana/pull/113157/files#diff-3844738ff7c0e0aa8a5e9f4dddb989764719b1c880f6571d36beb1a296aadcbdR617 right?

Reply:
Yep, artifact from when I was combining the attributes with the response. Thanks!